cloudinit/dmi.py

```diff
@@ -182,4 +183,42 @@ def is_x86(arch):
     return None
 
 
+def _get_dmi_keys():
```

Comment:
Related to my doc table comment, is there a reason we can't just match on keys and skip the values entirely?

Reply:
Dropped, we'll just use DMI standard keys.